adder.js

```diff
@@ -1,6 +1,6 @@
-function validate(validationField, email) {
+function validate(input) {
   let regex = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
-  let adress = document.forms[validationField].elements[email].value;
+  let adress = input.value;
```

Comment:
Please extract it out the function. Because our function should validate and argument should be email string. There shouldn't `input` here.

Reply:
> 
> 
> Please extract it out the function. Because our function should validate and argument should be email string. There shouldn't `input` here.

agree with @PFdev6